### ISSUE TITLE
Fix licence and contributing in Knowledge Base

### ISSUE THREAD

**Issue by yanesca** [Mbed-TLS/mbedtls#4074]
## Background

In https://github.com/ARMmbed/mbedtls/issues/4073 we are migrating Knowledge Base articles to the new, unified documentation site (https://github.com/ARMmbed/mbedtls-docs). Some of that information is out of date and we should at least fix the issues regarding licensing and contribution. Issues not related are out of scope, but quick easy wins might be included.

**Prerequisite:** https://github.com/ARMmbed/mbedtls/issues/4073

## Task

This task is done when the Knowledge Base articles in https://github.com/ARMmbed/mbedtls-docs are up to date regarding contribution and licencing.

**Comment by davidhorstmann-arm** [comment]
May be partly fixed by Mbed-TLS/mbedtls-docs#50.

**Comment by yanesca** [comment]
The issue has been fixed by fixed by https://github.com/Mbed-TLS/mbedtls-docs/pull/50.
